src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModel.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI
+{
+    public abstract class PackageModel
+    {
+        protected PackageModel(PackageIdentity identity, string title, string description, string authors, Uri projectUrl, string[] tags, DateTimeOffset? published)
+        {
+            if (identity == null)
+            {
+                throw new ArgumentNullException(nameof(identity));
+            }
+            Identity = identity;
+            Title = title;
+            Description = description;
+            Authors = authors;
+            ProjectUrl = projectUrl;
+            Tags = tags;
+            Published = published;
+        }
+
+        public PackageIdentity Identity { get; private set; }
+
+        public string Id
+        {
+            get => Identity.Id;
+        }
```

Comment:
More concise. This comment applies to the Version property as well.

```suggestion
        public string Id => Identity.Id;
```